### PR TITLE
Add tagline

### PR DESCRIPTION
**Pull request by ryancarlson**
A new field from name_description.xml for display on app home screen with each tool.

### CODE REVIEW

**Comment by frett** [comment]
I know we added support for arbitrary attributes to specific components. I can't remember if they were on Translation objects. If they were, we might be able to use that without needing an additional DB field to track it. Let me see if I can find where we defined them.

**Comment by frett** [comment]
nm, arbitrary attributes are at the resources level based on the API documentation

**Comment by ryancarlson** [comment]
@dsgoers can you review please?

File: db/schema.rb [hunk 1]
```diff
@@ -139,6 +139,7 @@
     t.string  "translated_name"
     t.string  "translated_description"
     t.string  "manifest_name"
+    t.string  "translated_tagline",     default: ""
```
Comment (dsgoers) [inline db/schema.rb]: Why did you add a default of empty string?

File: .rubocop.yml [hunk 2]
```diff
@@ -27,6 +27,10 @@ Documentation:
 Metrics/LineLength:
   Max: 120
 
+Metrics/ClassLength:
```
Comment (dsgoers) [inline .rubocop.yml]: I'd rather the class be refactored than silence the cop.

File: app/models/translation.rb [hunk 3]
```diff
@@ -112,6 +112,7 @@ def name_desc_onesky
     p = download_translated_phrases('name_description.xml')
     self.translated_name = p['name']
     self.translated_description = p['description']
+    self.translated_tagline = p['tagline']
```
Comment (dsgoers) [inline app/models/translation.rb]: Can you add a test for this?

File: .rubocop.yml (hunk 2 above)
Comment (ryancarlson) [reply .rubocop.yml]: It's over by one line, and I'm not experienced enough in Ruby/Rails yet to know how to refactor this well. My primary goal is to keep Greg moving

File: db/schema.rb (hunk 1 above)
Comment (ryancarlson) [reply db/schema.rb]: so that { translated_tagline: null } wouldn't come out in the JSON

File: app/models/translation.rb (hunk 3 above)
Comment (ryancarlson) [reply app/models/translation.rb]: There aren't any existing specs for this class that I could see...

File: db/schema.rb (hunk 1 above)
Comment (dsgoers) [reply db/schema.rb]: Well, it doesn't look like `translated_name` and `translated_description` have a default value...

File: app/models/translation.rb (hunk 3 above)
Comment (dsgoers) [reply app/models/translation.rb]: https://github.com/CruGlobal/mobile-content-api/blob/master/spec/models/translation_spec.rb#L157

File: .rubocop.yml (hunk 2 above)
Comment (dsgoers) [reply .rubocop.yml]: That makes sense, but silencing the check allows for the class to grow indefinitely.

File: app/models/translation.rb (hunk 3 above)
Comment (ryancarlson) [reply app/models/translation.rb]: ....
````
Offenses:

spec/models/translation_spec.rb:165:7: C: Example has too many expectations [3/2].
      it 'updates from OneSky' do
      ^^^^^^^^^^^^^^^^^^^^^^^^

130 files inspected, 1 offense detected

File: .rubocop.yml (hunk 2 above)
Comment (ryancarlson) [reply .rubocop.yml]: https://jira.cru.org/browse/GT-476

File: db/schema.rb (hunk 1 above)
Comment (ryancarlson) [reply db/schema.rb]: Does it hurt having a default value?

Comment (dsgoers) [reply db/schema.rb]: I don't think so.

File: .rubocop.yml (hunk 2 above)
Comment (dsgoers) [reply .rubocop.yml]: See my commit.

**Comment by ryancarlson** [comment]
good?
